### PR TITLE
Remove socialButtonStyle option to use small icons

### PR DESCRIPTION
**Pull request by luisrudge**
### Changes

Removes the `socialButtonStyle` option. This will remove the option to use `small` icons.

#### Before (`socialButtonStyle: 'small')`
image

#### After
image

### CODE REVIEW

**Comment by lbalmaceda** [comment]
Related to https://github.com/auth0/lock/issues/1510

File: src/ui/button/auth_button.jsx [hunk 1]
```diff
@@ -2,10 +2,9 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 const AuthButton = props => {
-  const { disabled, isBig, label, onClick, strategy, icon, primaryColor, foregroundColor } = props;
+  const { disabled, label, onClick, strategy, icon, primaryColor, foregroundColor } = props;
 
-  let className = 'auth0-lock-social-button';
-  if (isBig) className += ' auth0-lock-social-big-button';
+  let className = 'auth0-lock-social-button auth0-lock-social-big-button';
```
Comment (davidpatrick) [inline src/ui/button/auth_button.jsx]: className is never reassigned, should use const instead

Comment (luisrudge) [reply src/ui/button/auth_button.jsx]: fixed.

**Comment by victoriafrench** [comment]
omg this is ugly with multiple providers. there has got to be a better experience than a massive towering list.

**Pull request merged**